From e491478965fc9005362e44ba62c6b167d4d1b4b2 Mon Sep 17 00:00:00 2001
From: XScreenSaver owners <xscreensaver-owner@fedoraproject.org>
Date: Sun, 30 Dec 2018 14:45:54 +0900
Subject: [PATCH] fontglide.c: forbit C++ style comment

../../hacks/fontglide.c: In function 'fontglide_free':
../../hacks/fontglide.c:2424:1: error: C++ style comments are not allowed in ISO C90
 //  if (s->b && s->b != s->window) XFreePixmap (dpy, s->b);

Change comments with C style
---
 hacks/fontglide.c | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/hacks/fontglide.c b/hacks/fontglide.c
index df8f498..6c33d1a 100644
--- a/hacks/fontglide.c
+++ b/hacks/fontglide.c
@@ -2421,8 +2421,10 @@ fontglide_free (Display *dpy, Window window, void *closure)
 
   textclient_close (s->tc);
 
-//  if (s->b && s->b != s->window) XFreePixmap (dpy, s->b);
-//  if (s->ba && s->ba != s->b) XFreePixmap (dpy, s->ba);
+/*
+  if (s->b && s->b != s->window) XFreePixmap (dpy, s->b);
+  if (s->ba && s->ba != s->b) XFreePixmap (dpy, s->ba);
+*/
   XFreeGC (dpy, s->bg_gc);
   if (s->charset) free (s->charset);
   if (s->font_override) free (s->font_override);
-- 
2.20.1

